python/parpe/misc.py

```diff
@@ -315,15 +315,25 @@ class ParameterEstimationResultFile:
     pass
 
 
-def simulation_to_df(mes_df, sim, result_file, start, observable_ids):
+def simulation_to_df(mes_df, sim, result_file, start, observable_ids, input=False):
```

Comment:
```suggestion
def simulation_to_df(mes_df, sim, result_file, start, observable_ids, root_path: str = "/"):
```
?

Makes it more flexible.

Reply:
good idea, implemented it